src/Simplex/Chat/Types/Preferences.hs

```diff
@@ -152,6 +152,7 @@ data GroupFeature
   | GFSimplexLinks
   | GFReports
   | GFHistory
+  | GFNewMemberReview
```

Comment:
something feels wrong about it being a preference

Reply:
I am thinking about the time when member screening moves to chat relays, and while it has to be part of group profile, it feels like a separate space from preferences and requiring multiple toggles. At the same time it is unlikely to have role associated.

So we probably need profile property:

```
admission :: GroupMemberAdmission 

data GroupMemberAdmission = GroupMemberAdmission 
  { -- names :: AdmissionFilter,
    -- captcha :: AdmissionFilter,
    review :: AdmissionFilter
  }
  
 data AdmissionCriteria = ACAll | ACNoProfileImage -- etc. e.g. unverified profiles
```

effectively borrow the type that is now present in directory, and the whole admission process deserves a separate page, that initially would have one toggle.